less/index.less

```diff
@@ -1,59 +1,212 @@
-// Is this working?
-/* http://meyerweb.com/eric/tools/css/reset/ 
-   v2.0 | 20110126
-   License: none (public domain)
-*/
+@import "reset.less";
 
-html, body, div, span, applet, object, iframe,
-h1, h2, h3, h4, h5, h6, p, blockquote, pre,
-a, abbr, acronym, address, big, cite, code,
-del, dfn, em, img, ins, kbd, q, s, samp,
-small, strike, strong, sub, sup, tt, var,
-b, u, i, center,
-dl, dt, dd, ol, ul, li,
-fieldset, form, label, legend,
-table, caption, tbody, tfoot, thead, tr, th, td,
-article, aside, canvas, details, embed, 
-figure, figcaption, footer, header, hgroup, 
-menu, nav, output, ruby, section, summary,
-time, mark, audio, video {
-	margin: 0;
-	padding: 0;
-	border: 0;
-	font-size: 100%;
-	font: inherit;
-	vertical-align: baseline;
-}
-/* HTML5 display-role reset for older browsers */
-article, aside, details, figcaption, figure, 
-footer, header, hgroup, menu, nav, section {
-	display: block;
+/*===============================
+ * site variables
+ *===============================/
+
+/* color variables */
+@primary-color: #8c8c8c;
+@secondary-color: #000;
+@tertiary-color: #224e6a;
+@white: #fff;
+
+@hover-color: #05293f;
+
+// max width for website
+@maxWidthWebsite: 80%;
+
+/* background variables */
+@primary-bg-color: #e9ecef;
+@secondary-bg-color: #808080;
+
+/* border radius */
+@border-radius: 5px;
+
+/* container flow variables */
+.flex-baseline-between(@justifyContent) {
+  display: flex;
+  align-items: baseline;
+  justify-content: @justifyContent;
 }
-body {
-	line-height: 1;
+
+// base font sizes
+@defaultFontSize: 100%;
+@mobileFontSize: 150%;
```

Comment:
Not sure how would you use it. What is your thinking behind this?

Reply:
it was copied from the class lecture repo, I forget to remove it.